.github/workflows/ga_publish_ra_link.yml

```diff
@@ -0,0 +1,17 @@
+on: pull_request
+
+jobs:
+  example_comment_pr:
+    runs-on: ubuntu-latest
+    name: An example job to comment a PR
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v1
```

Comment:
This adds a comment everytime the job runs. We only really want one. Can we use a sticky comment instead as in https://github.com/DFE-Digital/teaching-vacancies/blob/main/.github/workflows/build_and_deploy.yml#L268

Reply:
ok i will look into this